custom-vu/src/main/kotlin/jces1209/vu/ScenarioSimilarities.kt

```diff
@@ -57,7 +54,8 @@ class ScenarioSimilarities(
             random = seededRandom,
             editProbability = 0.00f, // 0.10f if we can mutate data
             commentProbability = 0.00f, // 0.04f if we can mutate data
-            linkIssueProbability = 0.00f // 0.10f if we can mutate data
+            linkIssueProbability = 0.00f, // 0.10f if we can mutate data
+            attachScreenShotProbability = 0.00f
```

Comment:
I see multiple PRs changing the same places, which generates conflicts. One trick you can use is to add these params in random order. Same goes for other "common" files like `MeasureType` or `AbstractIssuePage`

In most cases the order of these lines doesn't matter.
And if it does matter, then conflicts are legit.